configure.ac

```diff
@@ -148,6 +148,8 @@ AX_CODE_COVERAGE()
 
 AX_CHECK_COMPILE_FLAG([-Wunknown-warning-option], [have_unknown_warning_option="yes"], [have_unknown_warning_option="no"])
 AM_CONDITIONAL([HAVE_UNKNOWN_WARNING_OPTION], [test "${have_unknown_warning_option}" = "yes"])
+AX_CHECK_COMPILE_FLAG([-Wshorten-64-to-32], [have_shorten_warning_option="yes"], [have_shorten_warning_option="no"])
+AM_CONDITIONAL([HAVE_SHORTEN_WARNING_OPTION], [test "${have_shorten_warning_option}" = "yes"])
```

Comment:
I will remove the shorten warning from the strict build. However, I'm not sure what you mean with the second part.

Reply:
the yaml changed since this comment and as I did not provide the permalink, it points to different place. The original line meant is this one:

https://github.com/OpenSC/OpenSC/blob/f0a3f67ab1ee3aa8e763859fbcddde79e1c3d411/.github/workflows/linux-strict.yml#L72

as we will not have the `CLANG_CFLAGS` anymore, this line can be simplified too.

Reply:
so I remove this line entirely!?

Reply:
Yes, I think that should do.